test/Directory.Build.props

```
@@ -3,9 +3,10 @@
 
   <PropertyGroup>
     <DefaultTestTargetFramework>netcoreapp2.1</DefaultTestTargetFramework>
+    <DefaultTestTargetFramework Condition="$(Configuration.Contains('-netfx'))">net462</DefaultTestTargetFramework>
```

Use, .EndsWith rather than Contains.
```suggestion
    <DefaultTestTargetFramework Condition="$(Configuration.EndsWith('-netfx'))">net462</DefaultTestTargetFramework>
```